index.html

```diff
@@ -0,0 +1,172 @@
+                        <img src="https://img.icons8.com/offices/30/000000/talk-male.png" alt="avatar"/>
+                    </span>
+                    <strong>Choose your language.</strong>
+                    <span class="lang-bar-header-close"> 
+                        <a href="#"> <img src="https://img.icons8.com/ios-glyphs/30/000000/multiply.png" alt="close"/> </a>
+                    </span>
+                </div>
+                <div class="lang-bar-content flex">
+                    <p>You're viewing YouTube in English (US). You can <span> <a href="#">change this preference below.</a> </span> </p>
+                    <a href="#"> <em>Learn More</em> </a>
+                </div>
+            </div>
+        </header>
+
+        <div class="mainContent  flex">
+            <main class="flex">
+                <iframe src="https://www.youtube.com/embed/rH1RTwaAeGc" 
+                 allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" 
+                allowfullscreen> </iframe>
+                <div class="iframe-discription -bottom">
+                    <h3>My Favorite VS Code Extensions</h3>
+                    <div class="iframe-discription-stats flex">
+                        <p>404,290 views</p>
+                        <p>Jan 8, 2019</p>
+                        <span>
+                            <a href="#"> <img src="https://img.icons8.com/material-rounded/24/000000/thumb-up.png" alt="like"/> 14K</a>
+                            <a href="#"> <img src="https://img.icons8.com/material-sharp/24/000000/thumbs-down.png" alt="dislike"/> 219</a> 
+                            <a href="#"> <img src="https://img.icons8.com/ios-glyphs/24/000000/forward-arrow.png" alt="sahre"/> SHARE</a>
+                            <a href="#">SAVE</a>
+                            <a href="#"> <img src="https://img.icons8.com/android/24/000000/more.png" alt="others"/> </a>
+                        </span>
+                    </div>
+                </div>
+                <div class="iframe-avatar flex">
+                    <picture class="avatar-img"> <img src="https://www.w3schools.com/howto/img_avatar.png" alt="avatar"/> </picture>
+                    <div class="avatar-disc">
+                        <strong>Colt Steele</strong>
+                        <p>115K subscribers</p>
+                        <p>Click "Show More" below for a full list of every extension I cover, as well as timestamps to jump ahead.</p>
+                        <a href="#">0:00:34</a> <span> Emmet</span>
+                        <br>
+                        <a href="#">SHOW MORE</a>
+                    </div>    
+                    <button>Subscribe</button>
+                </div>
+                <div class="loading  flex">
+                    <div class="loading-icon"> </div>
+                    <span>Loading  . . . </span>
+                </div>
+            </main>
+
+            <aside class="side-bar">
```

Comment:
Great use of semantic HTML 👌

- The title is sticking to the thumbnail preview.
![Annotation 2020-06-09 082849](https://user-images.githubusercontent.com/57726348/84101396-7a46d700-aa2b-11ea-8371-00255fcf280b.jpg)